NuGet.Config

```diff
@@ -3,7 +3,7 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-	<!-- Next line is causing problems with CI testing -->
+    <!-- Next line is causing problems with CI testing -->
```

Comment:
BecauseI really would prefer this is be here.
This makes it clear why it's not present, and hopefully at some point in the future I'll work out how to include this without breaking CI.